third_party/nvfuser/csrc/scheduler/matmul.cpp

```diff
@@ -325,4 +334,310 @@ void scheduleMatmul(
   }
 }
 
+void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
+  const auto& inputs = fusion->inputs();
+  const auto& outputs = fusion->outputs();
+
+  TORCH_CHECK(
```

Comment:
nit: Please use `TORCH_INTERNAL_ASSERT` instead of `TORCH_CHECK` for anything that's not directly called by users.

Reply:
I updated sources to use `TORCH_INTERNAL_ASSERT` in changed sources. I will follow this pattern in future changes, thanks!